core/scss/variables/core/_breakpoints.scss

```diff
@@ -242,60 +242,3 @@ $breakpoint-lg-only: 'only screen and (min-width: #{$screen-lg}) and (max-width:
 //
 // Styleguide: Variables.Core.Breakpoints.xl
 $breakpoint-xl: 'only screen and (min-width: #{$screen-xl})' !default;
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-//
-// $grid-xs
```

Comment:
None of these were used anywhere so I am assuming these are leftovers from the neat days.

Reply:
Indeed, they were.